### DISCUSSION TITLE
GESTIONE PRESENZE ALUNNI

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#403] [Idee e proposte]
Buonasera, 
Volevo sapere se fosse possibile implementare la gestione dell’appello, in particolare delle presenze degli alunni; ad esempio, mettendo “Presente Fuori Aula”, “Stage”, “Presente in altra attività”, e così via.
Qualora fosse complicato, sarebbe sufficiente inserire un campo di annotazione in fase di appello (a fianco alla schermata per selezionare presenze e assenze della classe) in cui il docente inserisce quanto detto sopra: mi rendo conto che può essere difficile inserire le altre tipologie di presenza, ma in questo modo credo sia più facile per lei che sviluppa il registro in quanto sarebbe “solo” da creare una casella di testo.

**Comment by trinko** [comment]
Purtroppo non è mail **solo** una casella di testo. Quello è solo per l'interfaccia, ma poi c'è la gestione dei dati sul database, la validazione e i controlli vari, ecc.

Comunque qualcosa di simile è già prevista nelle modifiche da apportare, ma prima devo concludere le altre modifiche.

**Reply by Lucaccc** [reply]
> Purtroppo non è mail **solo** una casella di testo. Quello è solo per l'interfaccia, ma poi c'è la gestione dei dati sul database, la validazione e i controlli vari, ecc.
> 
> Comunque qualcosa di simile è già prevista nelle modifiche da apportare, ma prima devo concludere le altre modifiche.

Sisi, mi rendo conto delle difficoltà.
Grazie per la dispnibilità.
